pom.xml

```diff
@@ -598,13 +598,42 @@
           <stagingProgressTimeoutMinutes>15</stagingProgressTimeoutMinutes>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
```

Comment:
can we use maven profiles instead?

`ant` seems so 1994

Reply:
done